type_templates/glb.js

```diff
@@ -565,10 +578,12 @@ export default e => {
             time: 0,
             animation: rideSpec.subtype,
             controllingId: instanceId,
+            controllingBone: rideSpec.sitBone ? rideBone : null,

```

Comment:
Does this need the `rideSpec.sitBone ?` ? Seems like that is covered above?

Reply:
Meaning, I think `rideBone` will already be `null` if we had no `sitBone`, so there is no need to check that condition again here.